### ISSUE TITLE
hasClippingOverflow(element) : Missing CSS overflow values

### ISSUE THREAD

**Issue by claudeG** [jamesshore/quixote#57]
Hi James !

Got "EnsureException: Unknown overflow property: **hidden auto**".
The problem is in descriptors/element_rendered_edge.js, function hasClippingOverflow(element).

No doubt that you know why.
For other readers : overflow-x is free to be different from overflow-y.

The complete set of standard values on one axis is { visible, hidden, clip, scroll, auto, overlay }.
'overlay' is deprecated and Quixote excludes it currently.
'clip' is not supported yet because it could be in the future.

In the first note of https://developer.mozilla.org/en-US/docs/Web/CSS/overflow, we can read:
"Setting one axis to visible (the default) while setting the other to a different value
results in visible behaving as auto."

So in hasClippingOverflow(element), the complete of values considered in the switch statement should be
(more legible in spreadsheet Excel or like. Duplicates are commented)

```javascript
   case "visible":         /* case "auto hidden": */  /* case "auto clip": */  /* case "auto scroll": */  /* case "auto": */
/* case "hidden auto": */     case "hidden":             case "hidden clip":      case "hidden scroll":      case "hidden auto":
/* case "clip auto": */       case "clip hidden":        case "clip":             case "clip scroll":        case "clip auto":
/* case "scroll auto": */     case "scroll hidden":      case "scroll clip":      case "scroll":             case "scroll auto":
/* case "auto": */            case "auto hidden":        case "auto clip":        case "auto scroll":        case "auto":
```

For what I understand in the current function code, all these cases return false and only "visible" returns true.

Regards

Claude G.

**Comment by claudeG** [comment]
Hi James !
Are you aware about any initiative related to the convertion of CAD-like
files to Quixote tests ?
Regards
Claude

**Comment by jamesshore** [comment]
No, I'm not aware of anything like that.

**Comment by claudeG** [comment]
While not completely adapted to the specification of UI parts, the
parametric draft mode of FreeCad allows to specify geometric constraints
very similar to those supported by Quixote. The result is an Xml file. So
it would not too difficult to generate test descriptors convertible to
Quixote. I see this as something very powerful. But FreeCad will become
frustrating very quickly. Need for a dedicated FreeCad plugin. Python
development... If you are inspired  : )

**Comment by jamesshore** [comment]
This is outside of the scope for Quixote.

**Comment by claudeG** [comment]
Possibly complementary but out of the scope I agree... Nevertheless
congratulations for your dev.
Regards
Claude

**Comment by jamesshore** [comment]
Fixed in v0.14.1.
